### PR TITLE
12.8.0 release (#5070)

### PR DESCRIPTION
**Pull request by mike-plummer**

### CODE REVIEW

**Comment by mjhenkes** [comment]
@mike-plummer the change log should be included in this pr too right?

**Comment by mike-plummer** [comment]
> @mike-plummer the change log should be included in this pr too right?

Thanks for the sanity check @mjhenkes , added the changelog

**Review by mjhenkes — APPROVED** [review]

File: docs/guides/references/changelog.mdx [hunk 1]
```diff
@@ -2,6 +2,72 @@
 title: Changelog
 ---
 
+## 12.8.0
+
+_Released 03/14/2023_
+
+**Features:**
+
+- The [Debug page](https://docs.cypress.io/guides/cloud/runs#Debug) is now able
```
Comment (emilyrohrbough) [inline docs/guides/references/changelog.mdx:11]:
```suggestion
- The [Debug page](/guides/cloud/runs#Debug) is now able
```
these all need to be relative links

**Review by emilyrohrbough — CHANGES_REQUESTED** [review]
need relative doc links

**Pull request merged**